assume/scenario/loader_pypsa.py

```diff
@@ -0,0 +1,232 @@
+# SPDX-FileCopyrightText: ASSUME Developers
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+import shutil
+from datetime import datetime, timedelta
+from pathlib import Path
+
+import pandas as pd
+import pypsa
+from dateutil import rrule as rr
+
+from assume import World
+from assume.common.forecasts import NaiveForecast
+from assume.common.market_objects import MarketConfig, MarketProduct
+
+
+async def load_pypsa_async(
+    world: World,
+    scenario: str,
+    study_case: str,
+    network: pypsa.Network,
+    marketdesign: list[MarketConfig],
```

Comment:
we should also keep this in line with other loader and load the market from the config file and not pass it manually